code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,221 @@
+	if(consumes > EVOLEVOLV && (!animal_count.Find(/mob/living/simple_animal/hostile/wendigo/alpha) || animal_count[/mob/living/simple_animal/hostile/wendigo/alpha] <= 0))
+		var/mob/living/simple_animal/hostile/wendigo/alpha/new_wendigo = new /mob/living/simple_animal/hostile/wendigo/alpha(src.loc)
+		new_wendigo.names = names
+		qdel(src)
+
+/mob/living/simple_animal/hostile/wendigo/alpha
+	desc = "You can't help but feel that, no matter what, you should have brought a bigger gun."
+	icon = 'icons/mob/giantmobs.dmi'
+	icon_state = "wendigo_noblood"
+	icon_living = "wendigo_noblood"
+	icon_dead = "wendigo_dead"
+	var/icon_enraged = "wendigo"
+	pixel_x = -16 * PIXEL_MULTIPLIER
+	health = 600
+	maxHealth = 600
+	speed = 5
+	move_to_delay = 10
+	melee_damage_lower = 35
+	melee_damage_upper = 50 //Jesus christ
+
+	attacktext = "pulverizes"
+	attack_sound = 'sound/weapons/heavysmash.ogg'
+	environment_smash_flags = SMASH_LIGHT_STRUCTURES | SMASH_CONTAINERS | SMASH_WALLS
+	var/enraged = 0
+	var/punch_throw_chance = 25
+	var/punch_throw_speed = 1
+	var/punch_throw_range = 10
+
+/mob/living/simple_animal/hostile/wendigo/alpha/Life()
+	..()
+	if(health < (maxHealth/2) && enraged == 0)
+		visible_message("<span class = 'warning'>\The [src] seems to slow down, but looks angrier.</span>","<span class = 'warning'>You're not sure what that sound was, but it didn't sound good at all.</span>")
+		enraged = 1
+		speed = 7
+		melee_damage_lower = 50
+		melee_damage_upper = 85
+		move_to_delay = 20
+		icon_state = icon_enraged
+		icon_living = icon_enraged
+		environment_smash_flags = SMASH_LIGHT_STRUCTURES | SMASH_CONTAINERS | SMASH_WALLS | SMASH_RWALLS
+		punch_throw_chance = 45
+		punch_throw_range = 15
+		punch_throw_speed = 3
+	if(health > 300 && enraged == 1)
+		enraged = 0
+		speed = initial(speed)
+		melee_damage_lower = initial(melee_damage_lower)
+		melee_damage_upper = initial(melee_damage_upper)
+		move_to_delay = initial(move_to_delay)
+		icon_state = initial(icon_state)
+		icon_living = initial(icon_living)
+		environment_smash_flags = initial(environment_smash_flags)
+		punch_throw_chance = initial(punch_throw_chance)
+		punch_throw_range = initial(punch_throw_range)
+		punch_throw_speed = initial(punch_throw_speed)
+
+/mob/living/simple_animal/hostile/wendigo/alpha/AttackingTarget()
+	..()
+	if(istype(target, /mob/living))
+		var/mob/living/M = target
+		if(punch_throw_range && prob(punch_throw_chance))
+			visible_message("<span class='danger'>\The [M] is thrown by the force of the assault!</span>")
+			var/turf/T = get_turf(src)
+			var/turf/target_turf
+			if(istype(T, /turf/space)) // if ended in space, then range is unlimited
+				target_turf = get_edge_target_turf(T, dir)
+			else
+				target_turf = get_ranged_target_turf(T, dir, punch_throw_range)
+			M.throw_at(target_turf,100,punch_throw_speed)
+
+/mob/living/simple_animal/hostile/wendigo/skifree
+	name = "yeti"
+	desc = "holding F might not work this time"
```

Comment:
capitalization, punctuation and ??? I guess this is a reference?

Reply:
You can hold F to go faster than the Skifree yeti